### PR TITLE
process legal archive items and remove its media

### PR DESCRIPTION
**Pull request by petrjasek**
only keep viewImage and only if that item is no longer in archived

### CODE REVIEW

File: server/ansa/commands/remove_expired_media.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+from datetime import datetime, timedelta
+from ansa.remove_expired_media import remove_expired_media
+
+
+class RemoveExpiredMediaCommand(superdesk.Command):
+
+    option_list = (
+        superdesk.Option('--days', dest='days', default=50),
+        superdesk.Option('--skip', dest='skip', default=0),
```
Comment (ride90) [inline server/ansa/commands/remove_expired_media.py:12]: I think we can use `type` argument for `skip`, `limit` and `days`

File: server/ansa/commands/remove_expired_media.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+            .skip(int(skip)) \
+            .limit(int(limit))
+
+        archived_service = superdesk.get_resource_service('archived')
+
+        i = 0
+        for item in cursor:
+            if item.get('associations') or item.get('renditions'):
+                i += 1
+                doc = item.copy()
```
Comment (ride90) [inline server/ansa/commands/remove_expired_media.py:32]: why do we need a copy here?

**Comment by ride90** [comment]
@petrjasek how we preserve `viewImage`?

**Comment by petrjasek** [comment]
> @petrjasek how we preserve `viewImage`?

that's in https://github.com/superdesk/superdesk/blob/8bf8a6615abaf38407818b9709437a6b10655696/server/ansa/remove_expired_media.py#L13

File: server/ansa/commands/remove_expired_media.py (hunk 2 above)
Comment (petrjasek) [reply server/ansa/commands/remove_expired_media.py]: that was copy on write..

**Pull request merged**